### PR TITLE
chore: Fixed doc link

### PR DESCRIPTION
**Pull request by yutachaos**
## Description
- Fixed document link for karpenter irsa.
## How Has This Been Tested?
- [ ] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [ ] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [ ] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Comment by bryantbiggs** [comment]
Thanks for the PR! This may be changing rather soon so lets hold off a bit:
- https://github.com/aws/karpenter/pull/1332
- https://github.com/aws/karpenter/issues/1574

**Review by bryantbiggs — APPROVED** [review]
ok link is still valid, thanks for the PR @yutachaos  - good to go @antonbabenko 👍🏽

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Some of the addon/controller policies that are currently supported include:
 - [EBS CSI Driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/example-iam-policy.json)
 - [VPC CNI](https://docs.aws.amazon.com/eks/latest/userguide/cni-iam-role.html)
 - [Node Termination Handler](https://github.com/aws/aws-node-termination-handler#5-create-an-iam-role-for-the-pods)
-- [Karpenter](https://github.com/aws/karpenter/blob/main/website/content/en/preview/getting-started/cloudformation.yaml)
+- [Karpenter](https://github.com/aws/karpenter/blob/main/website/content/en/preview/getting-started/getting-started-with-terraform/_index.md#create-the-karpentercontroller-iam-role)
```
Comment (antonbabenko) [inline README.md:50]:
```suggestion
- [Karpenter](https://karpenter.sh/preview/getting-started/getting-started-with-terraform/#create-the-karpentercontroller-iam-role)
```

Rendered website is nicer than Markdown sources, agree?

**Comment by antonbabenko** [comment]
This PR is included in version 18.20.2 :tada:

**Pull request merged**